oap-server/server-core/src/main/java/org/apache/skywalking/oap/server/core/query/EventQueryService.java

```diff
@@ -53,6 +56,14 @@ public Events queryEvents(final EventQueryCondition condition) throws Exception
         return getDao().queryEvents(condition);
     }
 
+    public Events queryEvents(final List<EventQueryCondition> conditions) throws Exception {
```

Comment:
Please be explicit about the condition. In the `List<EventQueryCondition>` case, should not have UUID. You should refactor to codes to make 2 `queryEvents` more clear.

Reply:
Copy.

Reply:
I can't see any change about this. Could you explain more?